src/styles/typography.css

```
@@ -2,12 +2,12 @@
   font-family: 'Nunito';
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Nunito" (font-family-name-quotes)

(font-family-name-quotes)